### DISCUSSION TITLE
jvm argument  for each connection how to setting?

### DISCUSSION THREAD

**Discussion by ccwxl** [dbeaver/dbeaver#11241] [Q&A] [answered]
Set a separate jvm parameter for each link. Different environments have different krb5.conf when using kerberos authentication, which needs to be specified with -Djava.security.krb5.conf

**Comment by kseniiaguzeeva** [comment, marked as answer]
Enterprise Edition http://dbeaver.com allows you to do it.
You have to open separate DBeaver instanses (with different workspaces) if you use Community Edition.
